lib/sunrise/sunrise.php

```diff
@@ -24,9 +24,10 @@ function network_not_found( $domain, $path ) {
 		'domain_requested' => $domain,
 		'path_requested'   => $path,
 	];
-	$data = json_encode( $data );
+	$data = wp_json_encode( $data );
```

Comment:
> json_encode() is discouraged. Use wp_json_encode() instead.

This should be safe: `wp_json_encode()` is defined in [`wp-includes/functoins.php`](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-includes/functions.php#L4135); that file is `require`'d [before](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-settings.php#L111) the system [loads](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-includes/ms-settings.php#L47) `sunrise.php` from [`ms-settings.php`](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-settings.php#L141)